### ISSUE TITLE
Rocket.Chat can't connect to self hosted server internally on mobile

### ISSUE THREAD

**Issue by chandlerlucius** [RocketChat/Rocket.Chat#24849]
When connecting to self hosted server via https url from inside the hosted network the web version can connect, but the mobile app cannot.

**Comment by Alison4kk** [comment]
in order to connect from the app, the server needs to be in a public domain and have a SSL certificate
I did it using NO-IP (noip.com)  and lets encrypt ( https://docs.rocket.chat/quick-start/installing-and-updating/rapid-deployment-methods/snaps/auto-ssl-with-snaps)

**Comment by chandlerlucius** [comment]
Correct. It has all that. You can connect to the server from outside the internal network, just not inside it.

**Comment by chandlerlucius** [comment]
Wondering if theres any updates on this ticket?

**Comment by sackoh** [comment]
I have a same problem here... please help us
